src/internal/m365/collection/exchange/handlers.go

```diff
@@ -29,7 +29,7 @@ type addedAndRemovedItemGetter interface {
 	GetAddedAndRemovedItemIDs(
 		ctx context.Context,
 		user, containerID, oldDeltaToken string,
-		cc api.CallConfig,
+		config api.CallConfig,
```

Comment:
request: lets standardize on a name and refactor all api.CallConfig in one full sweep, rather than piecemeal the renaming.  `config` is fine.  I like `cfg`, personally.  Feelings are not strong either way.